src/pypistats/__init__.py

```diff
@@ -329,7 +337,11 @@ def _percent(data: dict | list) -> dict | list:
     grand_total = _grand_total_value(data)
 
     for row in data:
-        row["percent"] = "{:.2%}".format(row["downloads"] / grand_total)
+        row["percent"] = (
+            "N/A"
+            if grand_total == 0
+            else "{:.2%}".format(row["downloads"] / grand_total)
+        )
```

Comment:
Can you give an example how to trigger getting "N/A"? The `max` change above fixes the original bug report.

Reply:
I like that linting rule. Ideally ruff would have flagged that for me automatically. Maybe this would be a separate PR, but it would be great to have docs for how to enable pre-commit for this repo. Would it make sense to `uv add ruff --dev` and the same with other dev dependencies, and then point devs to something like https://gist.github.com/zkurtz/4c61572b03e667a7596a607706463543 ?

Reply:
I don't know, we might hit another error before this. Let's leave this out.